go.mod

```diff
@@ -0,0 +1,20 @@
+module k8s.io/gengo
+
+go 1.13
+
+require (
+	fake v0.0.0
```

Comment:
drop this, it will be inherited by all dependants

Reply:
😬 right, ok.

Reply:
Done.